src/js/page_sections/Menu/Menu.js

```diff
@@ -81,7 +80,7 @@ class Menu extends Component {
   }
 
   render() {
-    const { allThemes } = this.props.navigation[this.props.intl.locale];
+    const { allThemes } = this.props.navigation;
 
     return allThemes.edges.length && (
```

Comment:
Unrelated to this PR, but could we make a cleanNavigation helper function, like those in js/helpers/cleanData.js so Menu.js isn't expecting data in a graphql specific format?
So instead of having to loop through things with edges and nodes could we just loop through a regular collection of objects? No need to nest it in allThemes either.